vagrant/README.md

```diff
@@ -0,0 +1,22 @@
+# Vagrant configuration
+This directory contains a vagrant configuration that sets up a development
+environment resembling the production configuration at LibreCores.org
+as closely as possible.
+
+## Customizing Vagrant configuaration
+If you need to customize Vagrant configuration, you can add a `Vagrantfile.private`
+file with your customizations.
+
+```ruby
+Vagrant.configure("2") do |config|
+    config.vm.synced_folder ".",  "/vagrant", id: "vagrant-root",type: 'sshfs'
+    config.vm.synced_folder "..", "/var/www/lc", id: "application", type: 'sshfs'
+    config.vm.define 'librecores' do |node|
+        node.vm.network :public_network, ip: '10.42.0.99'
+    end
+    config.vm.provider :virtualbox do |v|
+        v.cpus = 2
+        v.memory = 2048
+    end
+end
+```
```

Comment:
Could you either add this documentation to http://librecores-web.readthedocs.io/en/latest/devenv/quickstart.html (sources are in the docs folder of the source tree), or add at least a pointer to this README there?